app/notifications/process_client_response.py

```diff
@@ -81,10 +81,13 @@ def _process_for_status(notification_status, client_name, provider_reference):
         sent_by=client_name.lower()
     )
     if not notification:
-        current_app.logger.warning("{} callback failed: notification {} either not found or already updated "
-                                   "from sending. Status {}".format(client_name,
-                                                                    provider_reference,
-                                                                    notification_status))
+        current_app.logger.debug(
+            "{} callback failed: notification {} either not found or already updated from sending. Status {}".format(
```

Comment:
After separate logger message was made for duplicate updates, does this part of the message still make sense?  '...or already updated from sending'

Reply:
yeah we don't need this any more i think